backend/tests/test_webpage_parsing.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pytest
+import requests
+import src.tools.web_pages as wp
+
+TEST_PAGE_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), './test_page.html'))
+
+@pytest.fixture
+def local_test_page_url():
+    # Use file:// URL for local HTML file
+    return f'file://{TEST_PAGE_PATH}'
```

Comment:
Why not use the path here directly without defining a global variable? It is unlikely to be used by anything else.